app/Providers/HealthServiceProvider.php

```diff
@@ -64,7 +64,8 @@ public function boot(): void
             CacheCheck::new(),
             OptimizedAppCheck::new(),
             DatabaseCheck::new()
-                ->name('pgsql')
+                ->name('tenant')
+                ->connectionName('tenant')
```

Comment:
We let them set the `tenantConnectionName` in `SwitchTenantDatabasesTask` so doesn't this need to take that into account?

Reply:
It's actually hard coded to `tenant` in the config file now. But good point, I changed to to look at the config value.